src/sections/Resources.tsx

```diff
@@ -1,5 +1,298 @@
+                onClick={() => window.open(resource.url, "_blank")}
+              >
+                <img
+                  src={resource.thumbnail}
+                  alt={resource.title}
+                  className="absolute top-0 left-0 w-full h-3/5 object-cover opacity-50 transition-opacity duration-300 ease-in-out hover:opacity-100"
+                />
+                <div
+                  className="p-4 grid grid-cols-2 bg-black relative border-solid border-[1px] border-[#272727]"
+                  style={{
+                    position: "absolute",
+                    bottom: 0,
+                    width: "100%",
+                    height: "100%",
+                    zIndex: 2,
+                    background: `linear-gradient(to bottom right, #141515 30%, ${resource.color})`,
+                    transform: "translateY(0)",
+                    transition:
+                      "transform 0.3s ease-in-out, padding 0.3s ease-in-out",
+                  }}
+                >
+                  <div className="col-span-2">
+                    <div className="flex items-center pt-4 py-3">
+                      <div
+                        className="w-3 h-3 rotate-[-45deg]  border border-[#2962A5]"
+                        style={{ backgroundColor: resource.color, borderColor: resource.colorLight }}
+                      ></div>
+                      <p
+                        className="pl-1 sm:pl-3 text-xs"
+                        style={{ color: resource.colorLight }}
+                      >
+                        {resource.contentType}
+                      </p>
+                    </div>
+                    <h3 className="text-white col-span-2 text-xl">
+                      {resource.title}
+                    </h3>
+                  </div>
+                  <div className="col-span-2 flex items-end">
+                    <div className="flex justify-between items-center w-full">
+                      <div className="w-2/3">
+                        <p className="text-[#F5F3EF] text-xs">
+                          {resource.where}
+                        </p>
+                      </div>
+                      <div className="w-1/3">
+                        <p className="text-right text-xs" style={{color: resource.colorLight}}>
+                          {resource.date}
+                        </p>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          ))}
+        </div>
+
+        {!isSmallScreen && (
+          <div className="flex justify-center space-x-4 mt-12">
+            <button
+              onClick={previous}
+              disabled={currentIndex === 0}
+              className="flex flex-row justify-between items-center gap-1 border-solid border-[2px] border-[#3B3B3B] rounded-[12px] px-[16px] py-[5px] bg-[#1C1C1C] font-semibold"
```

Comment:
Damn, this is a lot of CSS. Can you make a common component for this as well as this is also repeating a lot. In this way, you just have to pass `onClick`, `label`, `icon`, `icon-position`, `color` and maybe a few other props only